src/EnergyPlus/UnitVentilator.cc

```diff
@@ -1136,7 +1155,44 @@ namespace UnitVentilator {
                         ErrorsFound = true;
-                    }
+                    } else {
+                        bool ExhastNodeNotFound = true;
+                        // check exhaust node
+                        for (NodeNum = 1; NodeNum <= state.dataZoneEquip->ZoneEquipConfig(CtrlZone).NumExhaustNodes; ++NodeNum) {
+                            if (state.dataUnitVentilators->UnitVent(UnitVentNum).AirInNode ==
+                                state.dataZoneEquip->ZoneEquipConfig(CtrlZone).ExhaustNode(NodeNum)) {
+                                ExhastNodeNotFound = false;
+                                break;
+                            }
+                        }
+                        // check induce node
+                        if (ExhastNodeNotFound) {
+                            bool InletNodeFound = false;
+                            for (NodeNum = 1;
+                                 NodeNum <=
+                                 state.dataZoneEquip->ZoneEquipConfig(state.dataUnitVentilators->UnitVent(UnitVentNum).ZonePtr).NumReturnNodes;
+                                 ++NodeNum) {
+                                InletNodeFound = ZonePlenum::ValidateInducedNode(state,
+                                                                                 state.dataUnitVentilators->UnitVent(UnitVentNum).AirInNode,
+                                                                                 state.dataZoneEquip->ZoneEquipConfig(CtrlZone).ReturnNode(NodeNum));
+                                if (InletNodeFound) break;
+                            }
+                            if (!InletNodeFound) {
+                                ShowSevereError(
+                                    state, RoutineName + CurrentModuleObject + "=\"" + state.dataUnitVentilators->UnitVent(UnitVentNum).Name + "\"");
+                                ShowContinueError(
+                                    state,
+                                    "..UnitVentilator inlet node name must be the same as either a zone exhaust node name or an induced "
+                                    "air node in ZonePlenum.");
+                                ShowContinueError(state, "..Zone exhaust node name is specified in ZoneHVAC:EquipmentConnections object.");
+                                ShowContinueError(state, "..Induced Air Outlet Node name is specified in AirLoopHVAC:ReturnPlenum object.");
+                                ShowContinueError(state,
+                                                  "..UnitVentilator inlet node name = " +
+                                                      state.dataLoopNodes->NodeID(state.dataUnitVentilators->UnitVent(UnitVentNum).AirInNode));
+                                ErrorsFound = true;
+                            }
+                        }
+                   }
                 }
```

Comment:
Allow UnitVentilator to accept an induced node of return plenum as an inlet node with ATMixer at the supply side, in addition to a zone exhaust node.